### ISSUE TITLE
Failing loading libraries on TraceLabWeb

### ISSUE THREAD

**Issue by cebernalc** [CoEST/TraceLab#17]
While TraceLab tries to load all the core libraries it has to resolve the dependencies of components implemented by the community. Moreover it uses the appDomin with all the libraries preloaded to resolve the dependencies of the components since those components require the TraceLabSDK among other libraries to work. This issue is triggered when it tries to load the assembly, although paths are correct it is not finding the dependencies nor the file itself. Here are 2 images from the debugging. the first one corresponds to the execution of the console application and the second one corresponds to the web application.

![image](https://cloud.githubusercontent.com/assets/5841200/24336082/43b2413a-1257-11e7-8e14-8c6477b07bbf.png)

![image](https://cloud.githubusercontent.com/assets/5841200/24336086/48e13396-1257-11e7-8a58-ed9ade507e98.png)

**Comment by cebernalc** [comment]
The types can not be resolved in the app domain. However I modified the creation of the domain in multiple ways changing the path of the libraries to real ones. The naturalness of the web apps generate an strange behavior when adding libraries at running time. As an example it tries to load the core libraries in another folder as shown in the image

![image](https://cloud.githubusercontent.com/assets/5841200/24336226/108c461e-1259-11e7-897f-8ede8e9cf41a.png)

**Comment by cebernalc** [comment]
This problem in summary was due to permissions on the folders that a web application can have access to.So, we tried different things to fix this issue, for the first one we put all components inside the binary folder of the web application and we changed the way TraceLab was extracting the paths to load the components. However, this worked for some components because the change of the paths was not consistent. As a second option we decided to mimic the structure of components that is on documents to put it inside the bin folder which gave us successful results in terms of loading some libraries from the components folder. Now that we are able to successfully load some libraries we need to fix another issue and figure out why some libraries are not working. I'm creating a separate issue for this
